src/kvstore/NebulaStore.cpp

```diff
@@ -387,23 +387,19 @@ void NebulaStore::asyncRemovePrefix(GraphSpaceID spaceId,
 
 
 ResultCode NebulaStore::ingest(GraphSpaceID spaceId,
-                               const std::string& extra,
-                               const std::vector<std::string>& files) {
+                               const std::string& extra) {
     decltype(spaces_)::iterator it;
     folly::RWSpinLock::ReadHolder rh(&lock_);
     RETURN_IF_SPACE_NOT_FOUND(spaceId, it);
     for (auto& engine : it->second->engines_) {
-        auto parts = engine->allParts();
         std::vector<std::string> extras;
+        auto parts = nebula::fs::FileUtils::listAllDirsInDir(extra.c_str());
         for (auto part : parts) {
+            auto path = folly::stringPrintf("%s/%s", extra.c_str(), part.c_str());
```

Comment:
maybe we should separate the different part to different engine, like nebula store init?

Reply:
Good Catch ! I missed